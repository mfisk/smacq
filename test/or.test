#!/bin/sh

bin/smacqq -m <<EOF 2>/dev/null
print -v, ts from pcapfile($TEST/0325@1112-snort.pcap) where (ts>0 || ts < 0) 
EOF
