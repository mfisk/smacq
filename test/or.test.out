ts = 954011609.360668
ts = 954011609.361618
ts = 954011609.362882
ts = 954011609.363329
ts = 954011728.020237
ts = 954011728.020780
ts = 954011728.021223
ts = 954011728.021391
ts = 954011728.022503
ts = 954011728.028445
ts = 954011728.102582
ts = 954011728.103886
ts = 954011728.120922
ts = 954011728.126184
ts = 954011728.129182
ts = 954011728.131837
ts = 954011728.134177
ts = 954011728.518317
ts = 954011728.518393
ts = 954011728.620389
ts = 954011728.620464
ts = 954011728.620554
ts = 954011728.620627
ts = 954011728.620701
ts = 954011729.019047
ts = 954011729.019123
ts = 954011729.119180
ts = 954011729.119256
ts = 954011729.119330
ts = 954011729.119403
ts = 954011729.119476
ts = 954011731.390280
ts = 954011731.393042
ts = 954011731.393132
ts = 954011731.393278
ts = 954011731.422224
ts = 954011731.422384
ts = 954011731.424355
ts = 954011731.430712
ts = 954011731.434037
ts = 954011731.435816
ts = 954011731.438199
ts = 954011731.439352
ts = 954011731.440841
ts = 954011731.449118
ts = 954011731.465803
ts = 954011731.473742
ts = 954011731.476108
ts = 954011731.477595
ts = 954011731.479025
ts = 954011731.480107
ts = 954011731.481168
ts = 954011731.483377
ts = 954011731.484744
ts = 954011731.485619
ts = 954011731.486659
ts = 954011731.487872
ts = 954011731.488971
ts = 954011731.490265
ts = 954011731.491501
ts = 954011731.492601
ts = 954011731.523056
ts = 954011731.524317
ts = 954011731.526237
ts = 954011731.527459
ts = 954011731.528634
ts = 954011731.529772
ts = 954011731.530822
ts = 954011731.532031
ts = 954011731.533792
ts = 954011731.535031
ts = 954011731.536216
ts = 954011731.537292
ts = 954011731.538362
ts = 954011731.539540
ts = 954011731.541175
ts = 954011731.542351
ts = 954011731.543539
ts = 954011731.544603
ts = 954011731.545646
ts = 954011731.546804
ts = 954011731.548553
ts = 954011731.549749
ts = 954011731.551027
ts = 954011731.552164
ts = 954011731.553174
ts = 954011731.554210
ts = 954011731.555230
ts = 954011731.768206
ts = 954011731.768626
ts = 954011731.774992
ts = 954011731.830889
ts = 954011731.890978
ts = 954011731.902515
ts = 954011731.914011
ts = 954011731.914651
ts = 954011731.915711
ts = 954011731.921672
ts = 954011731.922436
ts = 954011731.945492
ts = 954011734.970249
ts = 954011738.004917
ts = 954011741.048696
ts = 954011744.093205
ts = 954011747.127418
ts = 954011747.163434
ts = 954011750.182853
ts = 954011753.354234
